### DISCUSSION TITLE
Custom Moderation

### DISCUSSION THREAD

**Discussion by DeusDrizzyy** [NamelessMC/Nameless#2414] [Ideas]
**Add an option in StaffCP to be able to create custom thread moderation options, see an example of how it would work:**

- Be able to choose the name of the new moderation option.
- Choose a label that would be added to the topic if this moderation option was chosen.
- Choose which sub-forum the topic will be moved to.
- Choose whether you want to lock the topic.
- And if possible a standard response for the topic, for example: Your complaint was refused. (Answer would be posted in the thread by the person who clicked the moderation option.)

**Example:**
1° The player creates a thread to report another player.
2° The team member goes to the "Moderation" menu and clicks on the custom option configured in StaffCP, the topic is locked, receives the label "Resolved" and a message is posted on the forum in the name of the team member with a default message defined in StaffCP.

**Why add this?**
Most server owners who use NamelessMC to create forums just want to have an area for players to report bugs, make complaints, review punishment, etc. Adding this option to be able to customize the moderation menu would make life much easier for all server owners, make the forum more organized and facilitate moderation on the part of team members.

**Comment by PadowYT2** [comment]
doesnt https://namelessmc.com/resources/resource/43-forms-module/ do that?

**Comment by DeusDrizzyy** [comment]
> doesnt https://namelessmc.com/resources/resource/43-forms-module/ do that?

This is a form, I'm talking about the moderation button that is on every forum thread, read better.
